### PR TITLE
Add node and gnupg to run.yaml

### PR DESCRIPTION
**Pull request by SorenEricMent**
Known issue: NodeJS shell hang on input.

Signed-off-by: Yunchuan "Winslow" Hu <i@winsloweric.com>

### CODE REVIEW

**Review by StefanJum — APPROVED** [review]
Thanks, all good.
Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>

File: scripts/run.yaml [hunk 1]
```diff
@@ -14,3 +14,13 @@ runs:
...
     memory: 64
     networking: False
+  - name: node
+    rootfs: ../dynamic-apps/node
+    command: /bin/node
+    memory: 512
+    networking: True
+  - name: gnupg
+    rootfs: ../dynamic-apps/gnupg
+    command: /usr/bin/gpg
```
Comment (razvand) [inline scripts/run.yaml]: Add argument to `gpg` command for actual functionality.

**Review by razvand — CHANGES_REQUESTED** [review]
See my comments. Split PR in two commits, one for each application. Add an issue detailing the problem with NodeJS.

File: scripts/run.yaml (hunk 1 above)
Comment (SorenEricMent) [reply scripts/run.yaml]: I've added an issue about NodeJS hanging at https://github.com/unikraft/dynamic-apps/issues/79

For "Add argument to gpg command for actual functionality.", may I ask what do you mean by "actual functionality"? cuz gpg don't provide a shell and require user-defined arguments? Do you mean a --version argument?

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request merged**